feathr_project/feathr/_envvariableutil.py

```diff
@@ -59,5 +61,8 @@ def get_environment_variable(variable_key):
         password = os.environ.get(variable_key)
         if not password:
             logger.info(variable_key +
-                        ' is not set in the environment variables.')
+                        ' is not set in the environment variables. Try Key Vault value.')
```

Comment:
When an environment variable is not found, is the expectation for them to set the environment variable with keyVaultName so it is already set?
If it's latter, I suggest modifying the log message to ".....' is not set in the environment variables, fetching the value from Key Vault')

Reply:
updated as recommended.